clients/admin-ui/src/features/datastore-connections/datastore-connection.slice.ts

```diff
@@ -80,8 +157,7 @@ export const datastoreConnectionApi = createApi({
       Partial<DatastoreConnectionParams>
     >({
       query: (filters) => ({
-        url: CONNECTION_ROUTE,
-        params: mapFiltersToSearchParams(filters),
+        url: CONNECTION_ROUTE + mapFiltersToSearchParams(filters),
```

Comment:
This needed to be updated to manually create the query string because the `params` function provided by RTK Query is designed to create 1:1 mapped key/value pairs. It doesn't support the fastapi syntax of `?connection_type=mongodb&connection_type=sass`. It assumes there is 1 copy of each key.